homeassistant/components/mqtt/mixins.py

```diff
@@ -91,7 +91,11 @@
     async_subscribe_topics,
     async_unsubscribe_topics,
 )
-from .util import get_mqtt_data, mqtt_config_entry_enabled, valid_subscribe_topic
+from .util import (
+    get_mqtt_data,
+    mqtt_config_entry_enabled,
+    valid_subscribe_topic,
+)
```

Comment:
Why is this changed?

Reply:
Was caused by a previous commit and some `black` magic